Libraries/Components/View/PlatformViewPropTypes.ios.js

```diff
@@ -10,11 +10,6 @@
  * @flow
  */
 
-const Platform = require('Platform');
+import type {TVViewProps} from 'TVViewPropTypes';
 
-var TVViewPropTypes = {};
-if (Platform.isTVOS) {
-  TVViewPropTypes = require('TVViewPropTypes');
-}
-
-module.exports = TVViewPropTypes;
+export type PlatformViewPropTypes = TVViewProps | {};
```

Comment:
I'm not sure we need the `| {}` on this. Did it solve a particular problem for you or were you just trying to be consistent with the original code?

Reply:
Over consistency, I'll get rid of it, ty